src/sas/qtgui/Calculators/Shape2SAS/Tables/subunitTable.py

```diff
@@ -142,8 +142,8 @@ def hollow_sphere(self):
         """Return the hollow sphere dimensions"""
         name = {self.x: "R", self.y: "r"}
         units = {self.x: "Å", self.y: "Å"}
-        types = {self.x: "volume"}
-        bounds = {self.x: [0, inf]}
+        types = {self.x: "volume", self.y: "volume"}
+        bounds = {self.x: [0, inf], self.y: [0, inf]}
```

Comment:
I believe this is only relevant to generating plugins. Should it be in this PR? It is correct and wont' hurt